src/pages/Login/Login.js

```diff
@@ -1,5 +1,48 @@
+import React from 'react';
+import './Login.css';
+import { useNavigate } from 'react-router-dom';
+import Header from '../../components/Header/Header';
+import Card from '../../components/Card/Card';
+import Button from '../../components/Button/Button';
+import Input from '../../components/Input/Input';
+
 function Login() {
+    const navigate = useNavigate();
+    return(
+        <>
+        <Header />
+            <div className='mainContainer'>
+                <div className='cardContainer'>
+                    <Card content={
+                        <div className='cardContent'>
+                            <div className='cardText'>
+                                <h1 className='heading'>Taste<span>Buds</span></h1>
+                            </div>
+                            <div className='username'>
```

Comment:
you could add width and height to the input containers to make them taller and wider like it looks on figma